pages/common/verilator.md

```diff
@@ -0,0 +1,20 @@
+# verilator
+
+> Converts Verilog and SystemVerilog hardware description language (HDL) designs into a C++ or SystemC model that after compiling can be executed.
+> More information: <https://veripool.org/guide/latest/>.
+
+- Build the C project in the current directory:
+
+`verilator --binary -j 0 -Wall {{path/to/source.v}}`
+
+- Create C++ executable:
+
+`verilator --cc --exe --build -j 0 -Wall {{path/to/source.cpp}} {{path/to/output.v}}`
+
+- Perform linting over the code in the current directory:
+
+`verilator --lint-only -Wall`
+
+- Create XML output about the design (files, modules, instance hierarchy, logic and data types) to feed into other tools:
+
+`verilator --xml-output {{path/to/output.xml}} -Wall`
```

Comment:
```suggestion
`verilator --xml-output -Wall {{path/to/output.xml}}`
```

Optional change: unexpected option after placeholder: option can be moved one step back to conform this option specification:

```
           verilator --help
           verilator --version
           verilator --binary -j 0 [options] [source_files.v]... [opt_c_files.cpp/c/cc/a/o/so]
           verilator --cc [options] [source_files.v]... [opt_c_files.cpp/c/cc/a/o/so]
           verilator --sc [options] [source_files.v]... [opt_c_files.cpp/c/cc/a/o/so]
           verilator --lint-only -Wall [source_files.v]...
```